src/Core/Loading/ILoadedImage.cs

```diff
@@ -23,13 +23,37 @@
 
 namespace Reko.Core.Loading
 {
+    /// <summary>
+    /// This interface is exposed by loaded objects whose format Reko has discovered.
+    /// 
+    /// </summary>
     public interface ILoadedImage
     {
+        /// <summary>
+        /// The (absolute) file location from which this object was loaded.
+        /// </summary>
+        /// <remarks>
+        /// Reko abuses the 'file:' schema to model file paths within (possibly nested) archives.
+        /// This is accomplished by using the '#' character to separate path segments within the respective 
+        /// archives.
+        /// The syntax is as follows:
+        /// - A simple executable file test.dll in the folder c:\myprograms:
+        ///     file:///c:/myprograms/test.dll
+        /// - An executable file test.so inside a TAR archive:
+        ///     file:///home/username/archive.tar#lib/test.so
```

Comment:
This documentation should be update after `archive:` syntax introducing

Reply:
Fixed.